protocols/kad/src/protocol.rs

```diff
@@ -40,6 +40,9 @@ use std::{io, iter};
 use unsigned_varint::codec;
 use wasm_timer::Instant;
 
+/// The protocol name used for negotiating with multistream-select.
+pub const DEFAULT_PROTO_NAME: Cow<'static, [u8]> = Cow::Borrowed(b"/ipfs/kad/1.0.0");
```

Comment:
Since it's public, I would also do that:

```suggestion
pub const DEFAULT_PROTO_NAME: &'static [u8] = b"/ipfs/kad/1.0.0";
```